src/engines/tex.rs

```diff
@@ -21,29 +21,39 @@ pub enum TexResult {
     Errors,
 }
 
-
+#[derive(Debug,Default)]
 pub struct TexEngine {
     // One day, the engine will hold its own state. For the time being,
     // though, it's just a proxy for the global constants in the C code.
+
+    halt_on_error: bool,
+    initex_mode: bool,
+    synctex_enabled: bool,
 }
 
 
 impl TexEngine {
     pub fn new () -> TexEngine {
-        TexEngine {}
+        TexEngine::default()
```

Comment:
I think this will default `halt_on_error` mode to be false, correct? I want it to default to true. Is there a way to override the default for just that one field?

If not, we could always invert its meaning (`continue_on_error` or something).